packages/richtext-lexical/src/features/toolbars/shared/ToolbarDropdown/index.tsx

```diff
@@ -81,6 +82,7 @@ export const ToolbarDropdown = ({
   anchorElem: HTMLElement
   classNames?: string[]
   editor: LexicalEditor
+  group: { type: 'dropdown' } & ToolbarGroup
```

Comment:
Could you split up the `ToolbarGroup` into `ToolbarDropdownGroup` and `ToolbarButtonsGroup`? Then you can define `ToolbarGroup` as `ToolbarGroup =  ToolbarDropdownGroup | ToolbarButtonsGroup `and `ToolbarDropdownGroup` can easily be imported here.